submissions/christopherUshie/html-form/contactForm.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>The Contact form</h1>
```

Comment:
This h1 and form tags  should be indented very well and not be on same line vertically with body and html tags